### DISCUSSION TITLE
Feature request: font and text spacing

### DISCUSSION THREAD

**Discussion by songyang-dev** [open-webui/open-webui#5241] [Ideas]
**Is your feature request related to a problem? Please describe.**
I find it difficult to tell paragraphs apart when lines are always equally spaced in the chat responses and prompts.

**Describe the solution you'd like**
Let users set the line spacings and paragraph spacings.

**Describe alternatives you've considered**
No alternatives, maybe use ChatGPT instead? They have proper spacing in their UI.

**Comment by tjbck** [comment]
Should be fixed on dev!

**Reply by songyang-dev** [reply]
Good! I see it in the newest release. I used to open the source code and change the CSS myself
